src/dao/pages/DAOProposals/CreateDAOProposal.tsx

```diff
@@ -290,7 +290,9 @@ export function CreateDAOProposal() {
           title,
           description,
           linkToDiscussion = "",
-          nftTokenSerialId,
+          tokenType,
```

Comment:
do we need `tokenType` param based on `nftSerialId` we can also identify.?

Reply:
It would miss the edge case where nftserialId is eequal to Default SerialId which we pass for other type of transactions.